pyfstat/tcw_fstat_map_funcs.py

```diff
@@ -122,11 +184,36 @@ def get_maxF_idx(self):
         """
         return np.unravel_index(np.argmax(self.F_mn), self.F_mn.shape)
 
+    def get_logBtSG(self):
+        """Compute (natural log of the) transient-CW Bayes-factor B_tSG = P(x|HyptS)/P(x|HypG).
```

Comment:
Ahà! So its natural log @dbkeitel : Why not `ln`?

Reply:
Also... `HyptS`-->`Hyp[tS]`? `H_{tS}`?

Reply:
Copied like this from lalpulsar comments. Version with underscore (and mathrm) would be the proper texy one.